core.coffee

```diff
@@ -375,8 +375,11 @@
 						throw new Error('Either the url or resource must be specified.')
 					url = params.resource
 
-					if params.id?
-						url += '(' + escapeValue(params.id) + ')'
+					if params.hasOwnProperty('id')
+						if params.id?
```

Comment:
I've switched to the short-circuit method, as for `''` and `0` (@emirotin's version blocks `0`) they're both potentially valid ids, which is why I specifically check for nullish values